pkg/cmd/repo/rename/http.go

```diff
@@ -0,0 +1,40 @@
+package rename
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghinstance"
+	"github.com/cli/cli/v2/internal/ghrepo"
+)
+
+func runRename(client *http.Client, repo ghrepo.Interface, newRepoName string) error {
```

Comment:
Since the main function of this package is called `renameRun`, the fact that this function is called `runRename` is a bit confusing since the names are so similar and yet the functions do very different things.

Since this function only does the API request to rename the repo, maybe its name could reflect that? It could be apiRename or makeRenameRequest or something like that.